### ISSUE TITLE
No storage metrics from EX2300

### ISSUE THREAD

**Issue by AngelFreak** [czerwonk/junos_exporter#80]
I don't get any storage metrics from our Juniper EX2300-48P and EX2300-24T

There is nothing obvious in the log, and i get the other metrics just fine.
Im running junos_exporter Version: 0.9.5.

I have the feature enabled i the config:
```
features:
  environment: true
  bgp: false
  ospf: false
  isis: false
  nat: false
  l2circuit: true
  ldp: true
  routes: true
  routing_engine: true
  interfaces: true
  interface_diagnostic: true
  storage: true
  fpc: true
```

**Comment by pgporada** [comment]
@AngelFreak What is the output from your device when you call `show system storage` ?

**Comment by AngelFreak** [comment]
Here you go:
```
fpc0:
--------------------------------------------------------------------------
Filesystem              Size       Used      Avail  Capacity   Mounted on
/dev/gpt/junos          1.3G       809M       448M       64%  /.mount
tmpfs                   542M        44K       542M        0%  /.mount/tmp
tmpfs                   324M       540K       324M        0%  /.mount/mfs
```

**Comment by pgporada** [comment]
Aha, I suspect it's the following line that's throwing off the parsing for your device.
```
fpc0:
--------------------------------------------------------------------------
```

On my test boxes I receive this output
```
> show system storage 
Filesystem              Size       Used      Avail  Capacity   Mounted on
/dev/da0s1a             610M       179M       382M       32%  /
...
...
```

**Comment by czerwonk** [comment]
> Here you go:
> 
> ```
> fpc0:
> --------------------------------------------------------------------------
> Filesystem              Size       Used      Avail  Capacity   Mounted on
> /dev/gpt/junos          1.3G       809M       448M       64%  /.mount
> tmpfs                   542M        44K       542M        0%  /.mount/tmp
> tmpfs                   324M       540K       324M        0%  /.mount/mfs
> ```

Can you please provide the XML output of:

```
show system storage | display xml
```

Thanks in advance.

**Comment by AngelFreak** [comment]
Sure here you go:
```
<rpc-reply xmlns:junos="http://xml.juniper.net/junos/18.2R1/junos">
    <multi-routing-engine-results>
        
        <multi-routing-engine-item>
            
            <re-name>fpc0</re-name>
            
            <system-storage-information junos:style="brief">
                <filesystem>
                    <filesystem-name>/dev/gpt/junos</filesystem-name>
                    <total-blocks junos:format="1.3G">2796512</total-blocks>
                    <used-blocks junos:format="814M">1667792</used-blocks>
                    <available-blocks junos:format="442M">905000</available-blocks>
                    <used-percent> 65</used-percent>
                    <mounted-on>/.mount</mounted-on>
                </filesystem>
                <filesystem>
                    <filesystem-name>tmpfs</filesystem-name>
                    <total-blocks junos:format="537M">1099008</total-blocks>
                    <used-blocks junos:format="44K">88</used-blocks>
                    <available-blocks junos:format="537M">1098920</available-blocks>
                    <used-percent>  0</used-percent>
                    <mounted-on>/.mount/tmp</mounted-on>
                </filesystem>
                <filesystem>
                    <filesystem-name>tmpfs</filesystem-name>
                    <total-blocks junos:format="324M">663776</total-blocks>
                    <used-blocks junos:format="540K">1080</used-blocks>
                    <available-blocks junos:format="324M">662696</available-blocks>
                    <used-percent>  0</used-percent>
                    <mounted-on>/.mount/mfs</mounted-on>
                </filesystem>
            </system-storage-information>
        </multi-routing-engine-item>
        
    </multi-routing-engine-results>
    <cli>
        <banner>{master:0}</banner>
    </cli>
</rpc-reply>
```

**Comment by czerwonk** [comment]
Ok, the original storage module does not handle the multi-routing-engine-result Tag. I will fix this accordingly.
